client/homebrew/brewRenderer/brewRenderer.less

```diff
@@ -68,12 +68,16 @@
 @media print {
 	.toolBar { display : none; }
 	.brewRenderer {
-		height      : 100%;
-		padding-top : unset;
-		overflow-y  : unset;
+		height     : 100%;
+		padding    : unset;
+		overflow-y : unset;
+		&:has(.facing, .flow) {

```

Comment:
Just one question. Does this `:has()` rule add anything different than the `padding : unset;` above?

Reply:
Ah, I see it might be a specificity thing. If I remove the `:has` rule it doesn't work anymore because facing pages still have some other styling adding padding.

Reply:
You've got it - it's to revert the more specific styling to the original single page values during printing.